rdmo/management/assets/js/api/QuestionsApi.js

```diff
@@ -17,11 +17,15 @@ class QuestionsApi extends BaseApi {
     return this.get(url)
   }
 
-  static storeCatalog(catalog) {
+  static storeCatalog(catalog, action) {
```

Comment:
Maybe this could be a different function. `toggleSite(catalog, action)`?

Reply:
ok, I changed it up a bit now. 
Added a `MultiSiteApi.js` that has a general `UpdateElementAction` function, it's called in the `storeElement` via the `elementAction` prop. 
Is ok like that??

Reply:
I wrote a few thinks inline. I am still not super happy, but it makes sens to reuse the `storeElement` action. The Api Classes mimic the RDMO apps so `MultiSiteApi.js` does not really make sense. Maybe `SiteApi.js`. But as I have written elsewhere, maybe just add 3 tiny functions in the 3 classes.

Reply:
yes, Ive removed `MultiSiteApi.js` altogether